pulp_ansible/app/tasks/collections.py

```diff
@@ -146,45 +151,44 @@ def import_collection(artifact_pk):
         artifact_pk (str): The pk or the Artifact to create the Collection from.
     """
     artifact = Artifact.objects.get(pk=artifact_pk)
-    with tarfile.open(str(artifact.file.path), "r") as tar:
-        log.info(_("Reading MANIFEST.json from {path}").format(path=artifact.file.path))
-        file_obj = tar.extractfile("MANIFEST.json")
-        manifest_data = json.load(file_obj)
-        collection_info = manifest_data["collection_info"]
 
-        with transaction.atomic():
-            collection, created = Collection.objects.get_or_create(
-                namespace=collection_info["namespace"], name=collection_info["name"]
-            )
+    log.info("Processing collection from {path}".format(path=artifact.file.path))
+    importer_result = process_collection(str(artifact.file.path))
+    metadata = importer_result["metadata"]
```

Comment:
This was changed from `collection_info` to `metadata` since it was in our gdoc, but note the artifact key still says `collection_info` - galaxy-importer can use either term here